### DISCUSSION TITLE
is avalonia actually free to use on commerical project.

### DISCUSSION THREAD

**Discussion by jomarcenter-mjm** [AvaloniaUI/Avalonia#13258] [Q&A]
Would like to know as I am developing a open source mini game launcher solution (will be under MIT or similar licensing) that would be available for free for the public to use for commercial games and non-commercial games. and I want to make sure that the public won't have any trouble with licensing and such when using the software (esp since it would be marketing to the indie communities)

**Comment by timunie** [comment]
Yeah we offer paid support for companies. But you are not forced to have a support contract. Just add Avalonia to the Readme / Notice and you are fine. We ship with the MIT license which allows quite a lot of usage. 

Happy coding

**Reply by maxkatz6** [reply]
Side note, it's not necessary to mention Avalonia in the readme/notice, as MIT doesn't enforce it.
But it would be nice to mention it anyway.
